helm/examples/nfs/hpcc-nfs/values.yaml

```diff
@@ -30,3 +30,8 @@ planes:
   size: 3Gi
   labels: [ "data" ] # NB: all "data" planes will be auto mounted by engine components and others that require access to data
   rwmany: true
+- name: mydropzone
+  subPath: dropzone
+  size: 1Gi
+  rwmany: true
+  labels: [ "lz" ]
```

Comment:
I thought we were renaming labels here?

Reply:
Yes, I was going to do that in a separate pr.  I have a list of 2 or 3 things to rename + want to double check before I do.